ckanext/validation/jobs.py

```diff
@@ -117,18 +125,23 @@ def run_validation_job(resource):
          'validation_timestamp': validation.finished.isoformat()})
 
 
-def _validate_table(source, _format=u'csv', schema=None, **options):
+def _validate_table(source, _format='csv', schema=None, **options):
+
+    # TODO: Search if there is an equivalent way to use a proxy in Frictionless Framework v5
```

Comment:
I'd like this to be addressed before merging the change; it has security implications.

Reply:
Fixed handling the http_proxy correctly with frictionless framework v5 in [28608e9](https://github.com/frictionlessdata/ckanext-validation/pull/69/commits/28608e9deead5d8604874ba7db35eb8acb496bd9)

Reply:
👍